crates/bevy_text/src/text.rs

```diff
@@ -407,6 +407,30 @@ impl TextColor {
     pub const WHITE: Self = TextColor(Color::WHITE);
 }
 
+/// The background color of the text for this section.
+#[derive(Component, Copy, Clone, Debug, Deref, DerefMut, Reflect, PartialEq)]
+#[reflect(Component, Default, Debug, PartialEq, Clone)]
+pub struct TextBackgroundColor(pub Color);
+
+impl Default for TextBackgroundColor {
+    fn default() -> Self {
+        Self(Color::BLACK)
+    }
+}
+
+impl<T: Into<Color>> From<T> for TextBackgroundColor {
+    fn from(color: T) -> Self {
+        Self(color.into())
+    }
+}
+
+impl TextBackgroundColor {
+    /// Black colored text
+    pub const BLACK: Self = TextBackgroundColor(Color::BLACK);
+    /// White colored text
+    pub const WHITE: Self = TextBackgroundColor(Color::WHITE);
```

Comment:
The wording here is a little confusing IMO, we are talking about the background of the text, not the text itself.
```suggestion
    /// Black background
    pub const BLACK: Self = TextBackgroundColor(Color::BLACK);
    /// White background
    pub const WHITE: Self = TextBackgroundColor(Color::WHITE);
```

Reply:
Yep I copy and pasted these from `TextColor` and forgot to update the comments 😅